backend/nodes/image_filter_nodes.py

```diff
@@ -126,7 +126,11 @@ def __init__(self):
             ImageInput("Image"),
             ImageInput("Reference Image"),
             BoundedNumberInput(
-                "Reference Image Scale Factor", default=0.125, step=0.125
+                "Reference Image Scale Factor (%)",
+                minimum=0.1,
```

Comment:
Is there a reason for this minimum?

Same for `ImResizeByFactorNode`.

Reply:
Because 0 is an invalid scale number, but you could technically use a sub-1% scale (not sure why you'd want to, but you could).

Reply:
Didn't catch this originally, but yeah should probably make it like 0.00001 or something